### ISSUE TITLE
Display will change depending on tool selected

### ISSUE THREAD

**Issue by jkmarx** [refinery-platform/refinery-platform#1605]
- [x] Create a module for controlling to the tool panel display
    -Analyses and Visualizations will be separate modules
- [x] main ctrl
- [x] main factory (combined all tools into one obj)
``` 
{ Visualization: [ tool1, tool2, ... ], Analysis: [workflow1, workflow2, ....] } 
```
- [x] Main view (for tool directives/partials)
    - ~[ ] Tool panel can collapse~
    - ~[ ] Reformat UI Grid size if tool panel is open~ Moving into it's own issue.
- [x] Component & Partial for Tool Select Drop Down
- [x] Service for tracking selected tool

- [x] Unit Tests

**Comment by jkmarx** [comment]
https://www.bennadel.com/blog/2738-using-ngrepeat-with-nginclude-hurts-performance-in-angularjs.htm

**Comment by jkmarx** [comment]
Resolved by pull request #1642
